templates/head/fonts.twig

```diff
@@ -0,0 +1,4 @@
+<link rel="stylesheet" href="https://greenpeace.org/static/planet4-assets/SourceSans-Bold.woff2"/>
+<link rel="stylesheet" href="https://greenpeace.org/static/planet4-assets/SourceSans-BoldItalic.woff2"/>
+<link rel="stylesheet" href="https://greenpeace.org/static/planet4-assets/SourceSans-Italic.woff2"/>
+<link rel="stylesheet" href="https://greenpeace.org/static/planet4-assets/SourceSans3.woff2"/>
```

Comment:
The first 3 are 404 because you missed the 3 at the end of the font name :grin: (`SourceSans3`).

Also you can add `www` to the domain to avoid an extra reditect.